pages/common/manim.md

```diff
@@ -0,0 +1,36 @@
+# manim
+
+> Animation engine for explanatory math videos.
+> More information: <https://docs.manim.community>.
+
+- Render a scene from a Python script using the default settings:
+
+`manim {{path/to/file.py}} {{SceneName}}`
+
+- Render with live preview (auto-opens the video file after rendering):
+
+`manim -pql {{path/to/file.py}} {{SceneName}}`
```

Comment:
Can you look up if these short options have long variants?

Reply:
> Is it preferred to use the long ones throughout the page instead of the short ones
Yes, but you can documebt both with an option placeholder which allows the client to pick and choose what to display
```suggestion
`manim {{[-pql|--preview --quality low}}] {{path/to/file.py}} {{SceneName}}`
```